### PR TITLE
Remove passed pods from Add/Delete periodic update

### PR DESCRIPTION
**Pull request by Mmduh-483**

### CODE REVIEW

File: pkg/daemon/daemon.go [hunk 1]
```diff
@@ -247,26 +252,27 @@ func (d *daemon) AddPeriodicUpdate() {
...
 			if err != nil {
+				failedPods = append(failedPods, pod)
 				glog.Warningf("AddPeriodicUpdate(): failed to dump networks %+v of pod into json with error: %v",
 					networks, err)
 				continue
 			}
 			pod.Annotations[v1.NetworkAttachmentAnnot] = string(netAnnotations)
 			if err := d.kubeClient.SetAnnotationsOnPod(pod, pod.Annotations); err != nil {
 				if !strings.Contains(strings.ToLower(err.Error()), "not found") {
+					failedPods = append(failedPods, pod)
```
Comment (moshe010) [inline pkg/daemon/daemon.go]: if it not found it mean that it was deleted, no need to add it to the failedPods

File: pkg/daemon/daemon.go [hunk 2]
```diff
@@ -247,26 +252,27 @@ func (d *daemon) AddPeriodicUpdate() {
...
 					glog.Errorf("AddPeriodicUpdate(): failed to update pod annotations with err: %v", err)
 					continue
 				}
 
 				if err = d.guidPool.ReleaseGUID(guidList[index].String()); err != nil {
+					failedPods = append(failedPods, pod)
```
Comment (moshe010) [inline pkg/daemon/daemon.go]: also here

File: pkg/daemon/daemon.go [hunk 3]
```diff
@@ -376,13 +395,18 @@ func (d *daemon) DeletePeriodicUpdate() {
 			}
 		}
 
-		for _, guidAddr := range guidList {
+		for index, guidAddr := range guidList {
 			if err = d.guidPool.ReleaseGUID(guidAddr.String()); err != nil {
+				failedPods = append(failedPods, passedPods[index])
```
Comment (moshe010) [inline pkg/daemon/daemon.go]: what is passedPods? it seem weird that you add the passedPods to the failedPods

File: pkg/daemon/daemon.go (hunk 1 above)
Comment (Mmduh-483) [reply pkg/daemon/daemon.go]: I agree, the case here if it failed and it is not deleted

File: pkg/daemon/daemon.go (hunk 3 above)
Comment (Mmduh-483) [reply pkg/daemon/daemon.go]: The pods that are configured with IB and then passed the remove pkey in delete periodic, Then failed in releasing guid, I am not sure if the is case may happen,
do u want to discard this change?

File: pkg/daemon/daemon.go (hunk 2 above)
Comment (Mmduh-483) [reply pkg/daemon/daemon.go]: this is for removed pods, I agree it shouldn't fail here. removed.

File: pkg/daemon/daemon.go (hunk 1 above)
Comment (moshe010) [reply pkg/daemon/daemon.go]: ha ok I missed the not

**Pull request merged**